technical-reports/format/design-token.md

```diff
@@ -89,6 +100,7 @@ The value of the `$description` property MUST be a plain JSON string, for exampl
 {
```

Comment:
It's the still correct that `$value` is the only required property on a design token object. `$type` can still be omitted, as long as one of the parent groups has `$type`, or it's a reference to another token.

Reply:
@c1rrus I think it would be clearer to state that type is required along with name and value. Something like:

> Name, value and type are required. `$type` can be omitted from an individual token definition if defined on a parent group. See more information on Types below.

Reply:
@kevinmpowell If it's ok with you, I'd like to leave this as is and merge this PR given that @dbanksdesign & @ilikescience both approved it now.

I think this is a symptom of a broader issue that would be best addressed in a separate PR: The spec's current use of the word "property" is overloaded. There are:

- _Syntax_ properties - i.e. the `$whatever` properties that are written in the files.
- _Model_ properties (someone please suggest a better name! 😛) - the properties that the (parsed) tokens and groups have.

Perhaps we need to agree on terminology to differentiate between these two meanings and then apply that consistently throughout the whole spec.

I'd argue that the for the current statement that "$value is the only required property for a token" is correct in the _syntax_ sense. However, in the _model_ sense, every token has at least name, value and type.